app/js/renderer.js

```diff
@@ -301,13 +301,21 @@ MathJax.Hub.Register.StartupHook("AsciiMath Jax Ready", function () {
       'plantuml-server-url=http://www.plantuml.com/plantuml@']
     const href = window.location.href
     const fileName = href.split('/').pop()
-    let fileExtension = fileName.split('.').pop()
-    if (fileExtension !== '') {
+    attributes.push(`docfile=${fileName}`)
+    const fileNameExtensionPair = fileName.split('.')
+
+    if (fileNameExtensionPair.length > 1) {
+      let fileExtension = fileNameExtensionPair[fileNameExtensionPair.length - 1]
       // Remove query parameters
       fileExtension = fileExtension.split('?')[0]
       // Remove fragment identifier
       fileExtension = fileExtension.split('#')[0]
       attributes.push(`outfilesuffix=.${fileExtension}`)
```

Comment:
We should probably use a constant `.html` since the extension will convert AsciiDoc to HTML.
For reference, here's the definition: 

**outfilesuffix**
File extension of the output file (starting with a period) as determined by the backend (.html for html, .xml for docbook, etc.). (The value is not updated to match the file extension of the output file when one is specified explicitly). Safe to modify.

https://asciidoctor.org/docs/user-manual/#env-attributes


What do you think ?

//ping @mojavelinux

Reply:
changed to '.html'

Reply:
Thanks 👍